.circleci/config.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           command: pip install mock
       - run: 
           name: Install dependencies
-          command: pip install emoji
+          command: pip install numpy
```

Comment:
Whenever you add a new dependency to your project, be sure to update your `requirements.txt` file.

Reply:
Done!